django/db/backends/mysql/creation.py

```diff
@@ -75,13 +75,24 @@ def _clone_db(self, source_database_name, target_database_name):
         load_cmd[-1] = target_database_name
 
         with subprocess.Popen(
-            dump_cmd, stdout=subprocess.PIPE, env=dump_env
+            dump_cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, env=dump_env
         ) as dump_proc:
             with subprocess.Popen(
                 load_cmd,
                 stdin=dump_proc.stdout,
                 stdout=subprocess.DEVNULL,
+                stderr=subprocess.PIPE,
                 env=load_env,
-            ):
+            ) as load_proc:
```

Comment:
I guess it could be an opportunity to avoid the unnecessary double nesting of context managers by flattening them.